### PR TITLE
Add `SapperCommerce`, `Misiki Books` and `Litekart`

### PR DESCRIPTION
**Pull request by itswadesh**
Added 2 opensource projects and 1 website

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -193,6 +193,8 @@
 - [Statusfy](https://github.com/bazzite/statusfy) - Status page system using Tailwind CSS.
 - [Todolist](https://github.com/guillaumebriday/todolist-frontend-vuejs) - To-do list application using Tailwind CSS.
 - [LeagueStats](https://github.com/vkaelin/LeagueStats) - Statistics website for League of Legends players.
+- [SapperCommerce](https://github.com/itswadesh/sapper-ecommerce) - ecommerce using svelte & Tailwind CSS.
```
Comment (hawezo) [inline readme.md]: Can you change the description to be more explicit? Ecommerce what?

File: readme.md [hunk 2]
```diff
@@ -193,6 +193,8 @@
...
+- [Misiki Books](https://github.com/itswadesh/misiki-books) - Book shop using VueJS + Moltin + Tailwind CSS
```
Comment (hawezo) [inline readme.md]:
```suggestion
+ [Misiki Books](https://github.com/itswadesh/misiki-books) - Book shop using Vue + Moltin + Tailwind CSS.
```

File: readme.md (hunk 1 above)
Comment (joshmanders) [reply readme.md]: ping @tehpsalmist

**Comment by joshmanders** [comment]
ping @itswadesh

**Comment by itswadesh** [comment]
Hi. I have resolved the conflict as well as made the requested changes

File: readme.md [hunk 3]
```diff
@@ -197,6 +197,8 @@
 - [Statusfy](https://github.com/bazzite/statusfy) - Status page system using Tailwind CSS.
 - [Todolist](https://github.com/guillaumebriday/todolist-frontend-vuejs) - To-do list application using Tailwind CSS.
 - [LeagueStats](https://github.com/vkaelin/LeagueStats) - Statistics website for League of Legends players.
+- [SapperCommerce](https://github.com/itswadesh/sapper-ecommerce) - ecommerce storefront using svelte & Tailwind CSS.
```
Comment (hawezo) [inline readme.md]:
```suggestion
+ [SapperCommerce](https://github.com/itswadesh/sapper-ecommerce) - E-commerce storefront using Svelte & Tailwind CSS.
```

Comment (hawezo) [reply readme.md]: Can you update this to respect capitalization rules? After this change I think we can merge. :)

**Comment by itswadesh** [comment]
done

**Pull request merged**